src/soundsource.cpp

```diff
@@ -19,9 +19,10 @@
 #include "sound.hpp"
 #include "soundsource.hpp"
 
+using namespace std::literals::chrono_literals;
```

Comment:
Merge artifact? Either way, not needed, since `using namespace std::chrono_literals;` was used below.

Reply:
Right, but somehow the `using namespace` on old line 24 got deleted. Can just restore that. Basically, no changes needed to this file AFAICT.